assets/components/contributionsThankYou/contributionsThankYouPage.jsx

```diff
@@ -52,18 +51,16 @@ export default function ContributionsThankYouPage(props: PropTypes) {
 // ----- Auxiliary Components ----- //
 
 function BodyCopy(props: PropTypes) {
-  if (props.contributionType === 'ONE_OFF') {
-    return <ContributionsSurveySection />;
+  if (props.contributionType !== 'ONE_OFF') {
+    if (props.directDebit) {
+      return (
+        <div className="component-direct-debit-details__container">
+          <DirectDebitDetails {...props.directDebit} />
+        </div>
+      );
+    } else {
+      // recurring non-DD
+      return <EmailConfirmation />;
+    }
   }
-  // recurring
-  if (props.directDebit) {
-    return (
-      <div className="component-direct-debit-details__container">
-        <DirectDebitDetails {...props.directDebit} />
-        <ContributionsSurveySection />
-      </div>
-    );
-  }
-  // recurring non-DD
-  return <EmailConfirmation />;
```

Comment:
https://github.com/guardian/support-frontend/pull/1015/commits
I was having fun with the same file only the other week